TestApp/ios/Podfile.lock

```diff
@@ -25,8 +25,8 @@ EXTERNAL SOURCES:
 
 SPEC CHECKSUMS:
   AppCenter: 3cf7b9387549e9703dab068793e404013e8c0cc1
-  AppCenterReactNativeShared: 9b1141147dc3e69c7eb8bbaabbd530069f9f028e
+  AppCenterReactNativeShared: d96c34c7818e546eb817dbeadc7d532ed8bfa06a
 
 PODFILE CHECKSUM: 03eb6f496b055051ab9597bdf11ccbf1b7c950df
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.5.0
```

Comment:
@bmourat, please revert this line and consider update cocoapods to 1.5.3 on your local machine.